utils/resize_images.py

```diff
@@ -0,0 +1,66 @@
+"""
+This script resizes JPG and PNG images in a directory to a specified size
```

Comment:
Add a copyright statement, at least.  At some point, we will need a licence too.